.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "matching/submodules/kore"]
+	path = matching/submodules/kore
+	url = git@github.com:raduom/kore.git
```

Comment:
shouldn't this point to kframework/kore?

Reply:
I did think about that, but when my repository gets updated I can check it does not brake anything, so we get no real downtime. I could change to the kframework one though if anyone else wants to manage the possible braking scenarios.

Reply:
it won't break anything anyway because it will be pointed at a specific commit and we can take the time to update at our leisure.